docs/recipes/precompiling-with-webpack.md

```diff
@@ -0,0 +1,29 @@
+## Precompiling source files with webpack
+
+The AVA [readme](https://github.com/avajs/ava#transpiling-imported-modules) mentions precompiling your imported modules as an alternative to runtime compilation, but it doesn't explain how. This recipe shows how to do this using webpack.
```

Comment:
Add a link to this recipe in the AVA readme at https://github.com/avajs/ava#transpiling-imported-modules